pandas/tests/frame/methods/test_select_dtypes.py

```diff
@@ -88,6 +88,29 @@ def test_select_dtypes_exclude_include_using_list_like(self):
         e = df[["b", "e"]]
         tm.assert_frame_equal(r, e)
 
+    def test_select_dtypes_exclude_include_int(self):
+        df = DataFrame(
```

Comment:
can you add the issue number as a comment

Reply:
Ok